ZelBack/src/services/fluxCommunication.js

```diff
@@ -25,6 +25,10 @@ let numberOfFluxNodes = 0;
 
 const blockedPubKeysCache = new LRU(LRUoptions);
 
+const privateIpsList = ['192.168.', '10.', '172.16.', '172.17.', '172.18.', '172.19.',
```

Comment:
Are we sure of this list? Like where does it come from?

Reply:
https://www.techtarget.com/whatis/definition/private-IP-address

"Private IPv4 addresses have the following class configurations:

Class A IP addresses. Configurations range from 10.0.0.0 to 10.255.255.255. This class is for large networks and has 8 bits for the network and 24 bits for hosts.
Class B IP addresses. Configurations range from 172.16.0.0 to 172.31.255.255. This class is for medium networks and has 16 bits for the network and 16 bits for hosts.
Class C IP addresses. Configurations range from 192.168.0.0 to 192.168.255.255. This class is for smaller networks and has 24 bits for the network and 8 bits for hosts."

Reply:
Lots of sites refering the same information.